### PR TITLE
moved warnings array to outermost level of response json

### PR DESCRIPTION
**Pull request by ethanlabelle**

### CODE REVIEW

File: src/webserver/HttpBase.h [hunk 1]
```diff
@@ -430,7 +430,7 @@ handle_request(
         if (!dosGuard.add(ip, responseStr.size()))
         {
             warnings.emplace_back("Too many requests");
-            response["warnings"] = warnings;
+            result["warnings"] = warnings;
```
Comment (cjcobb23) [inline src/webserver/HttpBase.h]: Are warnings supposed to be inside the `result` object? I thought they were at the top level: https://xrpl.org/response-formatting.html

**Review by cjcobb23 — APPROVED** [review]
LGTM

**Review by natenichols — APPROVED** [review]

**Comment by cjcobb23** [comment]
Do we need to make this change for websocket as well?

**Comment by ethanlabelle** [comment]
The warnings array is in the correct spot in WsBase.h.

**Pull request merged**